### DISCUSSION TITLE
support of range break

### DISCUSSION THREAD

**Discussion by teathinker** [epezent/implot#343] [Ideas]
I am not sure if range break is supported.
A good use case is US stock market, which only open for 390 minutes of a trading day, so displaying all date time ticks will make chart very sparse. A good technique some other ploting framework uses is to have a range break which basically makes x axis from market close to next market open not displayed. Is there a easy way to do anything similar in ImPlot? Thanks.

**Comment by epezent** [comment]
This is not currently supported.

**Reply by teathinker** [reply]
> This is not currently supported.

Any plan to support in the future? Thanks.

I am thinking to add this feature, but I am not sure where to add it.
